packages/cozy-client/src/CozyClient.js

```diff
@@ -199,6 +199,7 @@ class CozyClient {
       event: TRIGGER_CREATION
     }
   ) {
+    const METADATA_VERSION = 1
```

Comment:
[very] minor and not blocking: I would have put this in it's own file with an export.

Reply:
At first, I searched for a `constants.js` file without success. But in the end, if we change something in `cozyMetadata`, we are likely to change `METADATA_VERSION` so I think it can stay here if you don't mind.